custom_components/alexa_media/__init__.py

```diff
@@ -554,6 +587,7 @@ async def async_update_data():
                     },
                 },
             )
+        return entity_state
```

Comment:
If we're changing the function scope, we should update the typing and docstring to explain what this does now and particularly why `entity_state` should be returned as opposed to the other values that are being updated here. If it's just a sign we've successfully updated, then perhaps a bool is appropriate. Or a dict with all values. I could be wrong as I wrote this years ago so maybe I was inconsistent before.

Reply:
Docstring updated.  I based this off the home assistant examples of DataUpdateCoordinator.  By returning the value here, it is automatically available to everything extending CoordinatorEntity(guard, temp, lights) without those needing a handle to the global data object.

Reply:
Ok makes sense. Thanks for that update then.